packages/react-button/src/common/isConformant.ts

```diff
@@ -0,0 +1,11 @@
+import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui/react-conformance';
+
+export function isConformant(testInfo: Omit<IsConformantOptions, 'componentPath'>) {
+  const defaultOptions = {
+    disabledTests: ['has-docblock', 'kebab-aria-attributes', 'as-*'],
+  };
+
+  const path = { componentPath: module!.parent!.filename.replace('.test', '') };
```

Comment:
It might be better to just include this in defaultOptions (same in other packages)